app/scripts/ui.js

```diff
@@ -248,6 +245,7 @@ async function start() {
 
     extensionPort.onMessage.addListener(messageListener);
     extensionPort.onDisconnect.addListener(resetExtensionStreamAndListeners);
+    initializeUiWithTab(activeTab);
```

Comment:
added because before we were waiting for the background connection to be active before starting the UI, now we load it without waiting.